portal-api/src/v1/middleware/headers/check-api-key.js

```diff
@@ -0,0 +1,42 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const { API_KEY } = process.env;
```

Comment:
This is using the same api key as `dtfs-central-api` and `external-api`. Just flagging to make sure that that's the correct approach